### ISSUE TITLE
Safe deletion of un-used branches

### ISSUE THREAD

**Issue by davidomarf** [daque-dev/viare#15]
There are some branches that aren't updated since 2+ months. Can we safely delete them?

Specially since tests are already deleted in develop version, and won't be implemented using previous libdaque version.

These are the test branches:

https://github.com/daque-dev/viare/tree/heightmap-perspective-test
https://github.com/daque-dev/viare/tree/perspective-test-completion

And this is another one named camera:

https://github.com/daque-dev/viare/tree/camera

**Comment by quevangel** [comment]
Check if they are merged. Remove if they are.

**Comment by quevangel** [comment]
After all. A branch is just a pointer to a commit.

**Comment by davidomarf** [comment]
I already deleted merged ones.  The ones left are all unmerged, and that's why I asked.

**Comment by quevangel** [comment]
Can't see the commit network in mobile. I will see them later.

**Comment by davidomarf** [comment]
Don't need to. 

In https://github.com/daque-dev/viare/branches/active you can see the active branches, and if they are in a PR or not.

**Comment by quevangel** [comment]
I could have made a merge without a PR.

**Comment by quevangel** [comment]
From what I can see perspective-test-completion is merged.

**Comment by davidomarf** [comment]
develop is ahead of both test branches. I remember those two being created before we started to use a develop branch. 

I suggest deleting them.

camera has one struct `Camera` that was never merged into develop: https://github.com/daque-dev/viare/compare/develop...camera

But it was created in Aug 19, previous date to the first useful tests.

I suggest deleting it, too.

I'll leave the merge or deletion to your consideration, since you created them. Maybe there's some work you'd like to keep.

**Comment by quevangel** [comment]
Then the tests **are** merged.

**Comment by davidomarf** [comment]
shh

**Comment by quevangel** [comment]
> Censored language.

**Comment by davidomarf** [comment]
It's branch deletion, then.
